### DISCUSSION TITLE
Smart collections or export filter as collection

### DISCUSSION THREAD

**Discussion by jhurlbut** [okio-ai/nendo-platform#20] [Ideas]
It would be great if you could create a dynamic collection based on a filter. Like a collection which is all tempo 120-125 and updates based on any changes to the library similar to smart playlists in other music apps. Also the ability to export a filtered collection to a new collection would be cool

**Comment by faradox** [comment]
Exporting a filter as collection is being implemented right now as https://github.com/okio-ai/nendo-web/pull/7 and https://github.com/okio-ai/nendo-server/pull/7

**Comment by faradox** [comment]
Smart collections are currently not planned but the idea might be revisited in the future.
